### ISSUE TITLE
Add RNCookieManagerIOS.h and RNCookieManagerIOS.m to your Xcode project

### ISSUE THREAD

**Issue by fernsheldon** [joeferraro/react-native-cookies#34]
So I did the following:

 - npm install react-native-cookies
- rnpm link react-native-cookies

Alls good here, no errors.  Then I added the following line to my index file:

import CookieManager from 'react-native-cookies';

and restarted my simulator... that's where I get this error

"Add RNCookieManagerIOS.h and RNCookieManagerIOS.m to your Xcode project"

It works fine, when the simulator is started from Xcode.

**Comment by shahen94** [comment]
+

**Comment by cbcye** [comment]
+1

**Comment by cbcye** [comment]
fix is issure, my problem is react-native-webview-bridge component has bug, make the build issure for this component, then uninstall bridge component is working.

**Comment by loveninteb** [comment]
+1

**Comment by himanshusoni** [comment]
+1

**Comment by himanshusoni** [comment]
Guys,
I resolved this issue by doing the following steps mentioned [react-native-linking-libraries](https://facebook.github.io/react-native/docs/linking-libraries-ios.html),

1. Open the xcode project, at <Project_Name>/ios/<Project_Name>.xcodeproject
2. Go to Project settings -> Build Settings -> Search Paths -> Header Search Paths
(highlighted as 4. in the image at the bottom of the document page mentioned above)
3. Look for RNCookieManagerIOS and make it as `recursive`

**Comment by valerybodak** [comment]
@himanshusoni 
Unfortunately, your solution not working for me... Help me please!

**Comment by RohanVB** [comment]
@himanshusoni the solution isn't working for me either - I tried manually linking it and also linking it automatically by the instructions [here](https://facebook.github.io/react-native/docs/linking-libraries-ios.html) with no luck on either fronts.

**Comment by duhufu1987** [comment]
I got the same red screen, the thing is, I'm building a RN project with expo,no XCode project or simulator was used.

**Comment by natterstefan** [comment]
@duhufu1987 Did you solve it? @thejacobseely did not fix my issue, too bad.

**Comment by duhufu1987** [comment]
@natterstefan  No, I've turned to use Asyncstorage for my project, which works.

**Comment by natterstefan** [comment]
@duhufu1987 okay, too bad. I need it to set&update cookies in a webview in the app. Do you maybe have a hint for me?

**Comment by duhufu1987** [comment]
@natterstefan  sorry, I've no idea

**Comment by karltaylor** [comment]
Still getting this error when running `react-native run-ios` but like @fernsheldon said, running from xcode works fine.

**Comment by ManigandanRaamanathan** [comment]
@fernsheldon any solution?

**Comment by nihp** [comment]
**Follow the steps**
============
```
1. Remove the node_modules
2. npm install
3. Run react-native link
4. Install the app again

```

**Check your project Xcode Libraries. If it not there in the Link binaries section add it manually and try to re-install the app.**

`"Add RNCookieManagerIOS.h and RNCookieManagerIOS.m to your Xcode project`

**Comment by pegasus1982** [comment]
Hi
I have faced same issue?
What will be solution for this issue?

**Comment by nihp** [comment]
@pegasus1982 Did you tried this

https://github.com/joeferraro/react-native-cookies/issues/34#issuecomment-485350720

What are the solutions you have tried
